### PR TITLE
PAYEE-791 - Fixing bug in code, see ticket for info

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: test/uk/gov/hmrc/tai/service/TaxCodeChangeServiceImplSpec.scala [hunk 1]
```diff
@@ -91,7 +104,7 @@ class TaxCodeChangeServiceImplSpec extends BaseSpec
               TaxCodeRecordFactory.createPrimaryEmployment(dateOfCalculation = previousCodeDate)
             )
           )
-          val taxCodeIncomes = Seq(baseTaxCodeIncome.copy(taxCode = "1185L"))
+          val taxCodeIncomes = Seq(baseTaxCodeIncome.copy(taxCode = "1285L"))
```
[the remark this answers is not in the document]
Comment (brendanstraughan) [reply test/uk/gov/hmrc/tai/service/TaxCodeChangeServiceImplSpec.scala:107]: This is to generate a taxCodeMismatch now that it is working correctly we actually have to have different tax codes

**Review by PerrettJ4 — APPROVED** [review]
Logic is sound to me and passing manual review.  

Well documented.

File: test/uk/gov/hmrc/tai/service/TaxCodeChangeServiceImplSpec.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (PerrettJ4) [reply test/uk/gov/hmrc/tai/service/TaxCodeChangeServiceImplSpec.scala:107]: Makes sense

**Review by Jake-Raffe — APPROVED** [review]

**Review by sergi-hmrc — APPROVED** [review]

**Review by vivekhmrc — APPROVED** [review]
LGTM 👍

**Pull request merged**